# SARS-CoV-2-Vars
Molecular and Genetic Evolutionary Analysis of African SARS-CoV-2 Variants to Inform Vaccine Efficacy

## Background
Towards the end of 2019, the Severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) emerged in Wuhan China. Since then the virus has spread all over the world including the African Continent. In Parallel a myriad of bioinformatics tools have been developed to analyze the insurmountable data generated to study the disease. Some tools are user friendly as they are created to run on graphical user interfaces (GUI) while others are a bit more technical requiring the users to have some expertise. The most popular GUI tool used often is Galaxy. In this project we aim to compare the Augur (a command line tool) and Galaxy as pertains to:
- their use
- their outputs
- their ability to carry out evolutionary analysis of SARS-CoV-2 

## Aim
- To find the evolutionary relationship between SARS-COV2 sequences from different countries in Africa using different phylogenetic analysis tools: Galaxy and Augur 

## Objectives
- To compare galaxy and augur workflows for the molecular and genetic evolutionary analysis of African SARS-CoV-2 variants
- To discover the mutations found in the different African SARS-CoV-2 variants

## Prerequisites
- Collect different SARS-CoV-2 from NCBI genome database originating from different regions in Africa.
- The tools used for the phylogenetic analysis of the dataset generated are both offline and online: Galaxy GUI and command-line tool, Augur. 
 
 ```conda install -c conda-forge -c bioconda augur```

## Workflow
<details>
 <summary>Galaxy Workflow</summary>
</details>
<details>
 <summary>Augur Workflows</summary>
</details>

## Contributors
1. [Martha Luka](https://github.com/marthaluka)
2. Zainab Elouafi
3. [Marion Nyaboke](https://github.com/marionnyaboke)
4. [Laurah Ondari](https://github.com/Nyasita)
5. [Leonard Uzairue](https://github.com/Uzairue1)
6. Sadaf
